env0/data_template.go

```diff
@@ -79,6 +79,16 @@ func dataTemplate() *schema.Resource {
 				Description: "The env0 application installation id on the relevant github repository",
 				Optional:    true,
 			},
+			"token_id": {
+				Type:        schema.TypeString,
+				Description: "The env0 application token id on the relevant gitlab account",
```

Comment:
the token_id is used also for Other vcs, so change the description accordingly

Reply:
WDYM?
Where we can use that for Other vcs?
can you suggest different description?

Reply:
something like:
`the token id used for private git repos or for integration with GitLab. More info: https://docs.env0.com/docs/templates#manage-git-connectivity`

Your call

Reply:
1e8ebd59